android/expoview/src/main/java/host/exp/exponent/fcm/ExpoFcmMessagingService.java

```diff
@@ -23,6 +23,6 @@ public void onMessageReceived(RemoteMessage remoteMessage) {
       return;
     }
 
-    PushNotificationHelper.getInstance().onMessageReceived(this, remoteMessage.getData().get("experienceId"), remoteMessage.getData().get("channelId"), remoteMessage.getData().get("message"), remoteMessage.getData().get("body"), remoteMessage.getData().get("title"), remoteMessage.getData().get("categoryId"));
+    PushNotificationHelper.getInstance().onMessageReceived(this, remoteMessage.getData().get("experienceId"), remoteMessage.getData().get("projectId"), remoteMessage.getData().get("channelId"), remoteMessage.getData().get("message"), remoteMessage.getData().get("body"), remoteMessage.getData().get("title"), remoteMessage.getData().get("categoryId"));
```

Comment:
`ExpoGcmListenerService` needs this change as well (it can still be used on the client side and apparently just uses the fcm implementation behind the scenes)